src/batch-client/index.ts

```diff
@@ -603,13 +603,19 @@ export class ZimbraBatchClient {
 		}).then(Boolean);
 
 	public downloadAttachment = ({ id, part }: any) =>
-		this.download({ id, part }).then(({ id, part, content }: any) => ({
+		this.download({ id, part, url:`/service/home/~/?auth=co&id=${id}&part=${part}` }).then(({ id, part, content }: any) => ({
```

Comment:
do we need to pass part? is it getting used in download method?

Reply:
I think download method can just return content and addition of id and part with that response can be handled in wrapper functions downloadAttachment and downloadDocument

Reply:
done , just send isSecure (were required) and url and receiving  content